tools/project_manager/add_files.php

```diff
@@ -383,7 +383,7 @@ function analyze()
             $row =& $this->page_file_table[$base];
 
             $error_msgs = '';
-            $warning_msgs = '';
+            $warning_msgs = array();
```

Comment:
I like this change. I would also like to have consistency in the file between `$error_msgs` and `$warning_msgs`. Could you add a second commit to this MR to change `$error_msgs` to use the same array-based syntax?

Reply:
Sure, sounds great.

Reply:
I haven't looked at this in detail and it's perhaps beyond the scope of this task but if you are keeping warnings and errors in arrays do you still need to keep a count of them in n_warnings etc.?

Reply:
@70ray, I was going to use your n_warnings advice, but warning_msg is per page (txt/image pair) whereas n_warnings is for all pages. So I still think it makes sense to track separately.